lib/importer/index.js

```diff
@@ -156,9 +156,9 @@ exports.Initialise = (config, router, prototypeKit) => {
     if (!header_names || header_names.length == 0) {
       response.error = {text: "Unable to detect header rows"}
     } else {
-      response.data = header_names.map((elem, index, _arr) => {
+      response.data = header_names.map((elem, index) => {
 
-        let examples = sheets_lib.GetColumnValues(session, index, cellWidth=20, count=5).inputValues
+        let examples = sheets_lib.GetColumnValues(session, index, /* cellWidth */ 20, /* count */ 5).inputValues
```

Comment:
Don't think I agree that this is better!

Reply:
Nor me, but I'm not sure the previous use was actually valid javascript, at least I can't find a description of using it that way